apps/devhub/tests/test_views_versions.py

```diff
@@ -186,6 +187,24 @@ def test_user_can_disable_addon(self, hide_mock):
         msg = entry.to_string()
         assert self.addon.name.__unicode__() in msg, ("Unexpected: %r" % msg)
 
+    @mock.patch('devhub.views.unindex_addons')
+    def test_user_can_unlist_addon(self, unindex):
+        self.addon.update(status=amo.STATUS_PUBLIC, disabled_by_user=False,
+                          is_listed=True)
+        res = self.client.post(self.unlist_url)
+        self.assertEqual(res.status_code, 302)
+        addon = Addon.with_unlisted.get(id=3615)
+        assert addon.status == amo.STATUS_PUBLIC
+        assert addon.is_listed == False
+
+        # Make sure we remove the addon from the search index.
+        assert unindex.delay.called
+
+        entry = ActivityLog.objects.get()
+        assert entry.action == amo.LOG.ADDON_UNLISTED.id
+        msg = entry.to_string()
+        assert self.addon.name.__unicode__() in msg, ("Unexpected: %r" % msg)
```

Comment:
You can of course provide the message part of the assert, but I usually don't bother, because pytest does the right thing and shows you what it was expecting, and what it got instead ;)